pages/reference/OS/updates/update-process.md

```diff
@@ -3,32 +3,107 @@ title: Update process details
 excerpt: Details of the {{ $names.os.lower }} self-service update process
 ---
 
-# What happens during a {{ $names.company.lower }} host OS update?
+# {{ $names.company.upper }} host OS updates
 
-The ability to remotely update the entire host operating system (OS) of your device, rather than just the user application, is a core feature of {{ $names.company.lower }}. Performing updates may feel like an opaque process, so we thought it would be useful to break it down.
+With OTA (Over-the-Air) functionality built in, {{ $names.company.lower }} enables you to remotely update the entire host operating system (OS) on your device, rather than just the user application. {{ $names.company.upper }} provides two methods to perform Host OS Updates (HUP):
```

Comment:
Make it clear that the user does not choose which type of update to run. Instead it's the nature of the update.

Reply:
thanks, added some notes regarding this